validator/client/service.go

```diff
@@ -250,3 +253,26 @@ func ConstructDialOptions(
 
 	return dialOpts
 }
+
+// fetch Graffiti from a file or flag.
+func fetchGraffiti(flag string) [][]byte {
+	if flag == "" {
+		// No graffiti.
+		return make([][]byte, 1)
+	}
+
+	data, err := ioutil.ReadFile(flag)
+	if err != nil {
+		// Single graffiti.
+		return [][]byte{[]byte(flag)}
+	}
+
+	// Multiple graffiti.
+	graffitiData := bytes.Split(bytes.Replace(data, []byte("\r\n"), []byte("\n"), -1), []byte("\n"))
+	// Shuffle the entries.
```

Comment:
Why? This isn’t documented anywhere in the flag description.

Reply:
I warned Ivan this was rough!  Yes, the operation when there are multiple lines of graffiti should be documented.